packages/firestore/test/integration/api/query.test.ts

```diff
@@ -61,6 +61,21 @@ apiDescribe('Queries', (persistence: boolean) => {
     });
   });
 
+  it('cannot issue limitToLast queries without explicit order-by', () => {
+    const testDocs = {
+      a: { k: 'a' },
+      b: { k: 'b' },
+      c: { k: 'c' }
+    };
+    return withTestCollection(persistence, testDocs, async collection => {
+      await expect(
+        collection.limitToLast(2).get()
+      ).to.be.eventually.rejectedWith(
+        'limitToLast() queries require specifying an orderBy() on at least one document field.'
```

Comment:
As stated above, this should throw an exception.

Reply:
Fixed.